connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1856,6 +2046,10 @@ ble_error_t Gap::setAdvertisingPayload(
     Span<const uint8_t> payload
 )
 {
+    tr_info("Set advertising payload for set %d", handle);
+#if TRACE_WRITE_VALUES
```

Comment:
I don't think we have to trace anything here, the trace can be in `setAdvertisingData` as this function just forward its call to it.

Reply:
Good point.